releasenotes/notes/ovn-23-09-1c710a01b00604b1.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    ``OVN`` version in Rocky Linux 9 container images has been updated to
+    ``23.09``.
```

Comment:
Looks like it affects both RL9 and Ubuntu, and also includes OVS 3.3.

Reply:
@mnasiadka tells me we already had OVS 3.3, this is just a more explicit pin.

Reply:
Well, I've rebuilt Ubuntu just for freshness, but Ubuntu/Antelope ovs is 3.1.x in UCA and ovn is 22.03 (from Jammy packages, not UCA)

Reply:
Ok, we'll need to consider how this interacts with @seunghun1ee's work to update all packages and container images.